Documentation/guides/k8s-aws-ansible/install.rst

```diff
@@ -0,0 +1,1430 @@
+    changed: [x.x.x.x]
+    changed: [x.x.x.x]
+    changed: [x.x.x.x]
+    
+    TASK [defunctzombie.coreos-bootstrap : Install pip launcher]     ***************************************************************************************************************
+    changed: [x.x.x.x]
+    changed: [x.x.x.x]
+    changed: [x.x.x.x]
+    changed: [x.x.x.x]
+    changed: [x.x.x.x]
+    changed: [x.x.x.x]
+    
+    PLAY RECAP **************************************************************************************************************************    ***************************************
+    x.x.x.x               : ok=7    changed=6    unreachable=0    failed=0   
+    x.x.x.x                : ok=7    changed=6    unreachable=0    failed=0   
+    x.x.x.x              : ok=7    changed=6    unreachable=0    failed=0   
+    x.x.x.x                : ok=7    changed=6    unreachable=0    failed=0   
+    x.x.x.x             : ok=7    changed=6    unreachable=0    failed=0   
+    x.x.x.x                : ok=7    changed=6    unreachable=0    failed=0   
+    
+    core@host:~/cilium/installation/ansible$ ansible -i hosts mps,sps -m shell -a "free"
+    x.x.x.x | SUCCESS | rc=0 >>
+                 total       used       free     shared    buffers     cached
+    Mem:       8178284     414336    7763948        328      12288     202236
+    -/+ buffers/cache:     199812    7978472
+    Swap:            0          0          0
+    
+    x.x.x.x | SUCCESS | rc=0 >>
+                 total       used       free     shared    buffers     cached
+    Mem:       8178280     412072    7766208        328      12308     202468
+    -/+ buffers/cache:     197296    7980984
+    Swap:            0          0          0
+    
+    x.x.x.x | SUCCESS | rc=0 >>
+                 total       used       free     shared    buffers     cached
+    Mem:       3857388     559156    3298232        340      35180     316372
+    -/+ buffers/cache:     207604    3649784
+    Swap:            0          0          0
+    
+    x.x.x.x | SUCCESS | rc=0 >>
+                 total       used       free     shared    buffers     cached
+    Mem:       3857388     557996    3299392        340      36384     316528
+    -/+ buffers/cache:     205084    3652304
+    Swap:            0          0          0
+    
+    x.x.x.x | SUCCESS | rc=0 >>
+                 total       used       free     shared    buffers     cached
+    Mem:       3857388     559832    3297556        340      36412     317296
+    -/+ buffers/cache:     206124    3651264
+    Swap:            0          0          0
+    
+    x.x.x.x | SUCCESS | rc=0 >>
+                 total       used       free     shared    buffers     cached
+    Mem:       8178284     411716    7766568        328      12320     201804
+    -/+ buffers/cache:     197592    7980692
+    Swap:            0          0          0
+
+
+3.2 TLS Assets 
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+::
+
+    core@host:~/cilium/installation/ansible$ 
```

Comment:
Needs a description that this verifies whether the keys have been copied properly. The output can be truncated to a single node with a note that says the output should be the same for each node